admin/shell.nix

```diff
@@ -0,0 +1,35 @@
+{
+  sources ? import ./npins,
+  system ? builtins.currentSystem,
+  pkgs ? import sources.nixpkgs { inherit system; },
+}:
+let
+  src = ./.;
+  python = pkgs.python3.withPackages (
+    ps: with ps; [
+      pandas
+      pandas-stubs
+      pydantic
+      githubkit
+      tqdm
+      ijson
+      pytest
+    ]
+  );
+  export-project-data = pkgs.writeShellScriptBin "extract-project-data" ''
+    ${pkgs.lib.getExe python} "${src}/scripts/export_project_data.py" "$@"
+  '';
+  sync-projects = pkgs.writeShellScriptBin "sync-projects" ''
+    ${pkgs.lib.getExe python} "${src}/scripts/sync_projects.py" "$@"
+  '';
+in
+pkgs.mkShellNoCC {
+  packages = [
+    export-project-data
+    python
+    sync-projects
+  ];
+  shellHook = ''
+    ln -s "${sources.ngipkgs}/projects/deliverables.py" scripts/lib/models/deliverables.py
```

Comment:
Got following message when running `nix-shell` 

```
ln: failed to create symbolic link 'scripts/lib/models/deliverables.py': File exists
```

Reply:
What about setting `PYTHONPATH` instead of creating symlink ? 

```
diff --git a/admin/shell.nix b/admin/shell.nix
index 9125e7a..d0f160f 100644
--- a/admin/shell.nix
+++ b/admin/shell.nix
@@ -29,7 +29,6 @@ pkgs.mkShellNoCC {
     python
     sync-projects
   ];
-  shellHook = ''
-    ln -s "${sources.ngipkgs}/projects/deliverables.py" scripts/lib/models/deliverables.py
-  '';
+
+  PYTHONPATH="${sources.ngipkgs}/projects";
 }
```
```
python -c "import deliverables; print(deliverables)"
<module 'deliverables' from '/nix/store/03g93s73x7i2izndxcnh51h5cdicl4s5-source/projects/deliverables.py'>
```

Reply:
Don't forget to remove scripts/lib/models/deliverables.py from the source code if you choose to use PYTHONPATH.

Reply:
That's a great alternative, thanks!